docs/configuration/config-file-reference.md

```diff
@@ -2329,6 +2329,22 @@ The `redis_config` configures the Redis backend cache. The supported CLI flags `
 # Enables connecting to redis with TLS.
 # CLI flag: -<prefix>.redis.enable-tls
 [enable_tls: <boolean> | default = false]
+
+# Close connections after remaining idle for this duration. If the value is
+# zero, then idle connections are not closed.
+# CLI flag: -<prefix>.redis.idle-timeout
+[idle_timeout: <duration> | default = 0s]
+
+# Enables waiting if there are no idle connections. If the value is false and
+# the pool is at the max_active_conns limit, the pool will return a connection
+# with ErrPoolExhausted error and not wait for idle connections.
+# CLI flag: -<prefix>.redis.wait
```

Comment:
Can we call it `wait-on-pool-exhaustion`? Similarly for the YAML option?

`-redis.wait` doesn't say much imo.

Reply:
Agree. Once done, please update the CHANGELOG accordingly.